src/NuGet.Clients/NuGet.SolutionRestoreManager/Resources.resx

```diff
@@ -124,7 +124,7 @@
     <value>Error occurred while restoring NuGet packages: {0}</value>
   </data>
   <data name="Error_PackageDownload_OnlyExactVersionsAreAllowed" xml:space="preserve">
-    <value>For package '{1}', '{0}' is not an exact version like '[1.0.0]'. Only exact versions are allowed with PackageDownload.</value>
```

Comment:
`Package is` sounds off to me. 
I prefer the package does not suggestion.

Reply:
If for example we are trying to PackageDownload "Nuget.Commands" with not an exact version like "v1", what we would get is `For package 'Nuget.Commands', 'v1' is not an exact version like '[1.0.0]'....` . I think you are right `Package 'Nuget.Commands' 'v1' does not have an exact version like [1.0.0]` sounds better.

Reply:
Another idea would be:

> Package 'Nuget.Commands v1' does not have an exact version like [1.0.0]

' ' is not allowed in either version or id, so it's good separator.